rmgpy/rmg/model.py

```diff
@@ -1343,31 +1343,13 @@ def getStoichiometryMatrix(self):
                 nu = rxn.getStoichiometricCoefficient(spec)
                 if nu != 0: stoichiometry[i,j] = nu
         return stoichiometry.tocsr()
-
-    def addSeedMechanismToCore(self, seedMechanism, react=False):
+    
+    def constructLibraryReactions(self,library):
         """
-        Add all species and reactions from `seedMechanism`, a 
-        :class:`KineticsPrimaryDatabase` object, to the model core. If `react`
-        is ``True``, then reactions will also be generated between the seed
-        species. For large seed mechanisms this can be prohibitively expensive,
-        so it is not done by default.
+        adds reactions from a given kinetics library in preparation to add it to core as seed mechanism
+        or to the edge as a reaction library
```

Comment:
I'm afraid I don't understand 1 it says in the docstring specifically that it is preparing the library to be added to the core as a seed mechanism or to the edge as a reaction library.  

2.  It is a library object.  

This is primarily a helper function for CoreEdgeReactionModel methods addSeedMechanismToCore and addReactionLibraryToEdge, (the only methods that call it).  It also couldn't be called without a CoreEdgeReactionModel to begin with so I really don't think it belongs outside of Model.py.